Kc dev

kelscahill opened the pull request.

kelscahill commented:
@designerbrent If all looks good, I will merge with master.

designerbrent commented:
@kelscahill Give me a few hours and I should have time to review this. I also think I need to get you the Favicons (or at least a starter set)

kelscahill commented:
@designerbrent Okay, great. No rush. =)

designerbrent commented:
I few notes, @kelscahill as I go through testing all these features. Not sure where to comment in the code since there is so much in this PR. (I prefer smaller, feature specific PR's).

- #126: I have updated the logos in included in the `dist` folder. I'll push those shortly.
- Author Hiding: I can't get this to work. You are looking for the variable `hide_author_post` in the post meta data, but I can't even see it in the DB myself. 
- #102: The `page` page types are showing on the edit page, but they don't show up on the front.
- #120: I'm not sure what was changed for this or how to test it.

designerbrent commented:
On the other items not mentioned in the code review, they look great. I'm really happy with how they are working.

kelscahill commented:
@designerbrent 

I have fixed the issues based on the PR comment. For #120, I just updated the `Single Full-Width Template' to hide the page title if it is set as the `front page`.
